routes/web.php

```diff
@@ -13,18 +13,33 @@
 |
 */
 
-Route::get('/', 'Home@index')->name('index');
+Route::get('/', 'HomeController@index')->name('index');
 
 Route::group(['prefix' => 'beta_keys', 'as' => 'beta_keys.'],
     function() : void{
-        Route::get('/', 'BetaKeys@index')->name('index');
+        Route::get('/', 'BetaKeysController@index')->name('index');
 
         Route::group(['prefix' => 'ajax', 'as' => 'ajax.'],
             function() : void{
-                Route::post('/create_key', 'Ajax\BetaKeysAjax@store')->name('create_keys');
-                Route::get('/get_key/{id?}', 'Ajax\BetaKeysAjax@get')->name('get_keys');
+                Route::post('/create_key', 'Ajax\BetaKeysControllerAjax@store')->name('create_keys');
+                Route::get('/get_key/{id?}', 'Ajax\BetaKeysControllerAjax@get')->name('get_keys');
 
             }
         );
     }
 );
+
+//TODO: Poprawić to by miało więcej sensu
```

Comment:
jesli uzywamy laravelowego Auth to chyba wystarczy dodać  Auth:;routes()

Reply:
Podeślij jakis link, ale ogólnie chodzi mi o czystość tego bloku